src/assets/Components/Album/Album.css

```diff
@@ -0,0 +1,147 @@
+.album-section {
+    display: grid;
+    grid-template-columns: 1fr;
+    padding-left: auto;
+    padding-right: auto;
+    justify-items: center;
+}
+
+.album-cards {
+    max-width: 300px;
+    padding: 20px;
```

Comment:
This styling together with the body's margin causes a tiny bit of sidescroll on mobile. Please fix 🛠️